stock_vertical_lift/models/vertical_lift_operation_pick.py

```diff
@@ -17,22 +17,37 @@ def _selection_states(self):
             ("scan_destination", "Scan New Destination Location"),
             ("save", "Pick goods and save"),
             ("release", "Release"),
+            ("skip", "Skip"),  # Virtual state.
         ]
 
     def _transitions(self):
         return (
             self.Transition(
                 "noop", "scan_destination", lambda self: self.select_next_move_line()
             ),
+            self.Transition("scan_destination", "skip", lambda self: self.is_skipped()),
             self.Transition("scan_destination", "save"),
+            self.Transition("save", "skip", lambda self: self.is_skipped()),
             self.Transition("save", "release", lambda self: self.process_current()),
+            self.Transition("release", "skip", lambda self: self.is_skipped()),
             # go to scan_destination if we have lines in queue, otherwise, go to noop
             self.Transition(
                 "release", "scan_destination", lambda self: self.select_next_move_line()
             ),
             self.Transition("release", "noop"),
+            self.Transition(
+                "skip",
+                "scan_destination",
+                lambda self: self.select_next_move_line(),
+                direct_eval=True,
+            ),
```

Comment:
Shouldn't you add `self.Transition("skip", "noop", direct_eval=True),` in case there is no more move line? When you skip, it should in normal case not happen as if you skip the last line, the same line will be re-opened, but if for any reason the line was the last, you skip it and for some reason it's no longer to be picked (deleted? done by someone else?), it returns to `noop`.